app/workers/field_tests/prune_old_experiments_worker.rb

```diff
@@ -6,7 +6,7 @@ class PruneOldExperimentsWorker
     def perform
       five_precent_membership_count = FieldTest::Membership.count / 20
       memberships = FieldTest::Membership.first(five_precent_membership_count)
-      FieldTest::Event.where(field_test_membership_id: memberships.pluck(:id)).delete_all
+      FieldTest::Event.where(field_test_membership_id: memberships.pluck(:id)).delete_all # rubocop:disable Rails/PluckInWhere
```

Comment:
Again not requesting the change but same suggestion as above, maybe `memberships.map(&:id)` can save us the rubocop disable annotation?